java/src/main/native/CMakeLists.txt

```diff
@@ -141,6 +141,18 @@ if(USE_GDS)
   endif()
 endif()
 
+# Set RMM logging level
+set(RMM_LOGGING_LEVEL
```

Comment:
@vyasr Is this the correct way to use rapids-logger? I discussed a bit with @abellina offline but I don't know the proper way to solve this problem.

Reply:
Assuming rmm is still being from source as part of the build, this will work, yes. It won't have any effect if you are using precompiled binaries, but since Spark doesn't do that this seems fine. The actual logs observed at runtime may still require setting the runtime log level if the logging functions are used.